src/org/mozilla/javascript/JavaMembers.java

```diff
@@ -918,3 +740,988 @@ public Object getDefaultValue(Class<?> hint)
+    {
+        //this.includeProtected = includeProtected;
+        try {
+            this.javaMemberScope = scope;
+            Context cx = ContextFactory.getGlobal().enterContext();
+            ClassShutter shutter = cx.getClassShutter();
+            if (shutter != null && !shutter.visibleToScripts(cl.getName())) {
+                throw Context.reportRuntimeError1("msg.access.prohibited",
+                                                  cl.getName());
+            }
+            this.members = new HashMap<String,Object>();
+            this.staticMembers = new HashMap<String,Object>();
+            this.cl = cl;
+            includePrivate = cx.hasFeature(
+                    Context.FEATURE_ENHANCED_JAVA_ACCESS);
+            //lazyInit = true;
+            cfCache = reflect(scope, includeProtected, includePrivate);
+        } finally {
+            Context.exit();
+        }
+    }
+
+    boolean has(String name, boolean isStatic)
+    {
+        if (cfCache.has(name, isStatic)) {
+            return true;
+        }
+        return findExplicitFunction(name, isStatic) != null;
+    }
+
+    Object get(Scriptable scope, String name, Object javaObject,
+               boolean isStatic)
+    {
+        Object member = getMember2(scope, name, isStatic);
+        if (member == null) {
+            member = this.getExplicitFunction(scope, name,
+                                              javaObject, isStatic);
+            if (member == null)
+                return Scriptable.NOT_FOUND;
+        }
+        if (member instanceof Scriptable) {
+            return member;
+        }
+        Context cx = Context.getContext();
+        Object rval;
+        Class<?> type;
+        try {
+            if (member instanceof BeanProperty) {
+                BeanProperty bp = (BeanProperty) member;
+                if (bp.getter == null)
+                    return Scriptable.NOT_FOUND;
+                rval = bp.getter.invoke(javaObject, Context.emptyArgs);
+                type = bp.getter.method().getReturnType();
+            } else {
+                Field field = (Field) member;
+                rval = field.get(isStatic ? null : javaObject);
+                type = field.getType();
+            }
+        } catch (Exception ex) {
+            throw Context.throwAsScriptRuntimeEx(ex);
+        }
+        // Need to wrap the object before we return it.
+        scope = ScriptableObject.getTopLevelScope(scope);
+        return cx.getWrapFactory().wrap(cx, scope, rval, type);
+    }
+
+    private Object getMember2(Scriptable scope, String name, boolean isStatic) {
+        Object member = getMember(scope, name, isStatic);
+        if (member == null && !isStatic) {
+            // Try to get static member from instance (LC3)
+            member = getMember(scope, name, true);
+            if( member == null) {
+                final Map<String,Object> ht = isStatic ? staticMembers : members;
```

Comment:
FWIW, "gradlew check" is failing on this line because "isStatic" is always false.

Reply:
> FWIW, "gradlew check" is failing on this line because "isStatic" is always false.

How to find the "gradlew check" detail report, I am   unfamilar for the "gradlew check"